<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntermediateClient</class>
 <widget class="QWidget" name="IntermediateClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Url">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>172.32.101.7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Connect">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>831</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_VehQueue">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>车队模式应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LandInitLoad">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>车道初始化应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GoWork">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上班应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_HoliDay">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>节假日模式应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_WorkOff">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下班应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QueueQuery">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>队列查询应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DownloadPara">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车道信息请求应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QueueAdd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>队列添加应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QueueDel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>队列删除应答</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UserInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据解密应答</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>连接状态</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Send">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>750</y>
     <width>1151</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1261</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>600</y>
     <width>210</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停接收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clean">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>750</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>10</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>831</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBox_GetBackCode_En">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>06-无车型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07-无车牌</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08-账号授权</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10-无称重信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12-车牌重复</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13-挂车称重确认</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14-车型车轴不符</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15-货车超限</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16-大件运输许可</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_CheckMsgCtrl">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确认信息回控请求</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SelectMsgCtrl">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>选项信息回控请求</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TextBroadCast">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>文字播音请求</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_HeartBeat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>心跳包</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ReadCard">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>01-上左读卡成功</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02-上右读卡成功</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03-下左读卡成功</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04-下右读卡成功</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05-上左读卡失败</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06-上右读卡失败</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07-下左读卡失败</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08-下右读卡失败</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_WriteCard">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>01-上左写卡成功</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02-上右写卡成功</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03-下左写卡成功</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04-下右写卡成功</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05-上左写卡失败</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06-上右写卡失败</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07-下左写卡失败</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08-下右写卡失败</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_PrepareCard">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>备卡-客1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-客2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-客3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-客4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-货1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-货2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-货3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-货4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-货5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-货6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-专1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-专2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-专3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-专4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-专5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备卡-专6</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>特情开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_End">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>特情结束</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TradeResult">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>交易结果</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_GetBackCode_Ex">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>01-车型不符</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02-车牌不符</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03-无入口信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04-车种不符</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05-车轴不符</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06-无车型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07-无车牌</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08-账号授权</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09-请求核单</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10-无称重信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11-拖车确认</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12-车牌重复</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16-大件运输许可</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_TradeInfo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>01-缴费信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02-车辆信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03-货车超限</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04-限行客车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05-限行危险品车</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
